health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AddOrEditMapping.js

```
@@ -0,0 +1,177 @@
+                            disabled={false}
+                            isMandatory={true}
+                            option={boundaryHierarchy}
+                            select={(value) => {
+                                setSelectedLevel(value);
+                                setSelectedBoundary(null);
+                            }}
+                            optionKey="boundaryType"
+                            t={t}
+                        />
+                        <div style={{height:"1rem"}}></div>
+                        <label>{t("CHOOSE_BOUNDARY")}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)